docs/examples/iq_plot_listen.py

```diff
@@ -0,0 +1,307 @@
+# Copyright 2020 SuperDARN Canada, University of Saskatchewan
+# Author: Kevin Krieger & Ashton Reimer
+"""
+Utilities for plotting and listening to DMAP IQ files
+
+Exceptions
+----------
+
+Functions
+---------
+iq_to_wav: Create a WAV file out of all records in the input iqdat file.
+Optionally use the I and Q signals to make a stereo WAV file (note: doubles the file size).
+The --play flag will call this if executing the script directly. The --stereo flag enables stereo.
+
+plot_iq: Create a plot of I and Q vs sample number for every record in the input iqdat file.
+The --plot flag will call this if executing the script directly.
+NOTE: This method takes a long time.  # TODO: HOW LONG?
+
+get_stats_from_record: Print out some useful stats about the input iqdat file. The --debug
+flag will call this if executing the script directly.
+
+open_compressed_file: General function that takes an input filename, and determines how to open
+it by reading the first few 'magic bytes' from the file. Returns a file stream acceptable to use
+with: pydarn.SDarnRead(filestream, True). The input iqdat file is opened with this method
+if executing this script directly.
+
+References: https://www.garykessler.net/library/file_sigs.html
+            https://superdarn.github.io/rst/superdarn/src.doc/rfc/0027.html
+"""
+import pydarn
+import binascii
+import numpy as np
+import datetime as dt
+import soundfile
+import matplotlib.pyplot as plt
+import argparse as ap
+import os
+
+
+def iq_to_wav(records, save_directory, sequences_to_play=None, stereo=False):
+    """
```

Comment:
pydarn uses numpy/pandas style docstring: https://numpydoc.readthedocs.io/en/latest/format.html

Should be an easy update.